README.md

```diff
@@ -44,5 +71,6 @@ in possession of the ball and rotates positions clockwise.
 
 *Note: This is meant to be fun, don't be a douche. When in doubt, redo the play, as the ball does not lie.*
 
-## End of Game:
-At the conclusion of the game, players of both sides line up and shake hands. This is an expected and time-honored tradition.
+* If a player hits three consecutive lets, then on the third let within a single possession will count as live.
```

Comment:
I think this sentence is wonky. Maybe that last chunk could be `then the third let will count as live.`